README.md

```diff
@@ -65,6 +65,31 @@ By default GitHub investigation results are displayed in the CLI itself. But you
 
 </details>
 
+
+<details>
+<summary>Investigate and update OpsGenie Issues with findings</summary>
+
+```bash
+holmes investigate opsgenie --opsgenie-api-key <PLACEHOLDER_APIKEY>
+```
+
+By default OpsGenie investigation results are displayed in the CLI itself. But you can use `--update --opsgenie-team-integration-key <PLACEHOLDER_TEAM_KEY>` to get the results as a comment in the OpsGenie issue.
+
+</details>
+
+
+<details>
+<summary>Investigate and update PagerDuty Issues with findings</summary>
+
+```bash
+holmes investigate pagerduty --pagerduty-api-key <PLACEHOLDER_APIKEY>
+```
+
+By default PagerDuty investigation results are displayed in the CLI itself. But you can use `--update --pagerduty-user-email <PLACEHOLDER_EMAIL>` to get the results as a comment in the PagerDuty issue.
```

Comment:
Is it worth adding a screenshot here (and for opsgenie) showing where the note from Holmes can be found in PagerDuty?